lib/routes/landiannews/index.ts

```diff
@@ -1,145 +1,27 @@
-                    const categories: string[] = [...new Set(categoryEls.map((el) => $$(el).attr('title') ?? $$(el).text()).filter(Boolean))];
-                    const authorEls: Element[] = $$('span.author a[title]').toArray();
-                    const authors: DataItem['author'] = authorEls.map((authorEl) => {
-                        const $$authorEl: Cheerio<Element> = $$(authorEl);
-
-                        return {
-                            name: $$authorEl.attr('title') ?? '',
-                            url: $$authorEl.attr('href') ?? '',
-                            avatar: $$authorEl.find('img.avatar').attr('src'),
-                        };
-                    });
-                    const upDatedStr: string | undefined = pubDateStr;
-
-                    let processedItem: DataItem = {
-                        title,
-                        description,
-                        pubDate: pubDateStr ? timezone(parseDate(pubDateStr, 'YYYY年M月D日 HH:mm'), +8) : item.pubDate,
-                        category: categories,
-                        author: authors,
-                        content: {
-                            html: description,
-                            text: description,
-                        },
-                        updated: upDatedStr ? timezone(parseDate(upDatedStr, 'YYYY年M月D日 HH:mm'), +8) : item.updated,
-                        language,
-                    };
-
-                    const extraLinkEls: Element[] = $$('section.article-list a[title]').toArray();
-                    const extraLinks = extraLinkEls
-                        .map((extraLinkEl) => {
-                            const $$extraLinkEl: Cheerio<Element> = $$(extraLinkEl);
-
-                            return {
-                                url: $$extraLinkEl.attr('href'),
-                                type: 'related',
-                                content_html: $$extraLinkEl.text(),
-                            };
-                        })
-                        .filter((_): _ is { url: string; type: string; content_html: string } => true);
-
-                    if (extraLinks) {
-                        processedItem = {
-                            ...processedItem,
-                            _extra: {
-                                links: extraLinks,
-                            },
-                        };
-                    }
-
-                    return {
-                        ...item,
-                        ...processedItem,
-                    };
-                });
-            })
-        )
-    ).filter((_): _ is DataItem => true);
+    const items: DataItem[] = await fetchNewsItems(postApiUrl);
 
     return {
-        title: $('title').text(),
-        description: $('meta[property="og:description"]').attr('content'),
-        link: targetUrl,
+        title: '蓝点网',
+        description: '给你感兴趣的内容!',
+        link: String(rootUrl),
```

Comment:
`rootUrl` https://github.com/DIYgod/RSSHub/blob/ff54862e2a5143044edb6f81787079cd51112220/lib/routes/landiannews/index.ts#L5 is already a string. No need to cast it as a string again.